### PR TITLE
Start locally linked example with npm scripts

### PR DESCRIPTION
**Pull request by rolandszoke**

### CODE REVIEW

File: scripts/startExample.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+);
+console.customInfo('Linking react-easy-state.');
+exec('npm link react-easy-state', {
+  cwd: exampleFolder,
+  stdio: 'inherit',
+});
+
+// 3. START EXAMPLE
+console.customInfo(`Starting ${process.argv[2]} in the background.`);
+exec("nohup bash -c 'npm run start' >/dev/null 2>&1 &", {
```
Comment (solkimicreb) [inline scripts/startExample.js]: This swallows the logs from create-react-app which are really valuable during development. I see why it was needed though. Any ideas how to keep the current behavior while also showing cra logs?

File: scripts/startExample.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+const gracefullShutdown = () => {
+  console.customInfo('Killing nodes.');
+  exec('killall node', { stdio: 'inherit' });
+  console.customInfo('Unlinking react-easy-state.');
+  exec('npm unlink --no-save react-easy-state', {
+    cwd: exampleFolder,
+    stdio: 'inherit',
+  });
+  exec('npm unlink', { stdio: 'inherit' });
```
Comment (solkimicreb) [inline scripts/startExample.js]: I am not 100% sure about this one. This removes the global symlink from npm which kills all other npm links the user might have for easy-state. I think that removing the local link (`npm unlink --no-save react-easy-state'`) is enough here. What do you think?

File: scripts/startExample.js (hunk 1 above)
Comment (rolandszoke) [reply scripts/startExample.js]: I'll look into it :)

Comment (rolandszoke) [reply scripts/startExample.js]: I'll look into it :)

**Comment by solkimicreb** [comment]
Thanks! This is pretty cool 😎

**Pull request merged**